dependency-graph/tests/view/dependency_test.txt

```diff
@@ -0,0 +1,5 @@
+8
+7
+11
+12
+10
```

Comment:
either hardcode.

keep whole scenario in a single file

Reply:
resolved!